homeassistant/components/zyxelt50/router.py

```diff
@@ -0,0 +1,287 @@
+""" The Zyxel T50 router """
+""" I used these as a starting point: https://github.com/ThomasRinsma/vmg8825scripts """
```

Comment:
This should probably be implemented in an external library

Reply:
yep, see: <https://developers.home-assistant.io/docs/development_checklist/>

Reply:
I've created an external library for the modem and use that now.